server.go

```diff
@@ -707,6 +712,32 @@ func (s *Server) Start() {
 		}
 	}
 
+	s.lastFlushUnix = time.Now().UnixNano()
+
+	// Be a watchdog forever!
+	go func() {
+		if s.stuckIntervals == 0 {
+			// No watchdog needed:
+			return
+		}
+		// if we haven't flushed in 3x the flush interval, we're stuck.
+		ticker := time.NewTicker(s.interval)
+		for {
+			select {
+			case <-s.shutdown:
+				ticker.Stop()
+				return
+			case <-ticker.C:
+				last := time.Unix(0, atomic.LoadInt64(&s.lastFlushUnix))
+				if time.Since(last) > 3*s.interval {
```

Comment:
Is there a reason for `3` here?

Reply:
formerly hardcoded, I should use the config value.